### PR TITLE
Adds a bigger chunk of documentation

### PR DESCRIPTION
**Pull request by funkyfuture**
ciao, i'd be glad to get some feedback from some experienced eXist-db users and also from unbiased people.

the text is supposed to evoke interest in, well, the targeted audience is probably somewhat similar to us. but doesn't have all that magic available yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,106 @@
...
+
+## Some supplemental architectural notes
+
+All installed instances are recorded in a central `csv` file. It is used by the
+mentioned `existctl` script and can be used for more tooling like monitoring as
+directory. It only holds the instances' name, id and the maximal allocatable
+memory for the JVM - name and id must never be changed.
+
+A central paradigm is that an instance's id is used as the port that its Jetty
+is listening to. Hence the restriction of available ports is a transitive
```
Comment (funkyfuture) [inline README.md]: it should be made clearer that 'available ports' refers to TCP/UDP.

File: README.md [hunk 2]
```diff
@@ -19,31 +111,108 @@ all keys that are presented in this sample:
...
 
+# the default -XmX value for new instances
 XmX_default = 1024m
 
+# this list contains names of Jetty configuration files that are not to be 
+# used, e.g. because a modern web server can do the job for all instances
 unwanted_jetty_configs = jetty-ssl.xml,jetty-ssl-context.xml,jetty-https.xml
 ```
 
+There are still many opinionated values hardcoded in the tool respectively the
```
Comment (funkyfuture) [inline README.md]: ~opinionated~

File: README.md [hunk 3]
```diff
@@ -19,31 +111,108 @@ all keys that are presented in this sample:
...
+the instance's dashboard is available at https://exist.mydomain.web/my_project/
+and you're good to go.
+
 ### uninstall
 
+This is basically the opposite of the previous and you can get rid of an 
+instance as easy as you type:
+
+    existance uninstall --id <id>
+
```
Comment (schnoepf) [inline README.md]: Maybe it is useful to save the data directory?

File: README.md [hunk 4]
```diff
@@ -19,31 +111,108 @@ all keys that are presented in this sample:
...
 ### upgrade
```
Comment (schnoepf) [inline README.md]: Some versions need backups of the data an reimport it to the modernized system. Maybe a usecase for the upgrade mechanism?

File: README.md (hunk 3 above)
Comment (funkyfuture) [reply README.md]: that'd be a discussion on features, not the docs reflecting these.

File: README.md (hunk 4 above)
Comment (funkyfuture) [reply README.md]: as above?

**Pull request merged**